### DISCUSSION TITLE
Pynecone Docs sidebar: Leaf / Reference item collapsing (mis)behavior

### DISCUSSION THREAD

**Discussion by gsal** [reflex-dev/reflex#559] [General]
Hey, guys: 

This is of no consequence; but, I just wanted to bring to your attention a bit of an annoying behavior in the sidbar of Pynecone documentation...I am talking about that behavior where 
- when one clicks on a child of a different item (I am using the terms as in sources pcweb/components/sidebar.py), the previously expanded item is automatically collapsed; and,
- when one clicks on a differernt child of the same item, nothing else collapses and the page on the right simply switches contents

This works ok in the "Learn" section; but not in the Reference section.

In the Reference section, whenever a click on a component, say, Components->Forms->Button, everything collapses and I lose my place!  

...just saying.

**Comment by Alek99** [comment]
Thanks for pointing this out. This should be fixed now lmk if its not working for you

**Reply by gsal** [reply]
Yeap, that part is fixed.  Thanks.
